task-sdk/pyproject.toml

```
@@ -231,4 +237,5 @@ tmp_path_retention_policy = "failed"
 [tool.airflow]
 shared_distributions = [
     "apache-airflow-shared-timezones",
+    "apache-airflow-shared-secrets_masker",
```

```suggestion
    "apache-airflow-shared-secrets-masker",
```

Given one of the above comments

---

Yep! Handled in https://github.com/apache/airflow/pull/54449/commits/635b21dbd8076269425bf4cf72ec83a6a08191f0